ifrs17/DataModel/DataStructure.ipynb

```diff
@@ -2154,6 +2155,115 @@
             "metadata": {},
             "execution_count": 0,
             "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Variable for Recording the Logs"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "The following variables are defined in order to keep the file information and keep trace of the logs, each time we access the DB."
```

Comment:
Why DB? Shouldn't this track all activities?

Reply:
Renamed